ui/tests/helpers/flash-message.js

```diff
@@ -0,0 +1,3 @@
+import FlashObject from 'ember-cli-flash/flash/object';
+
+FlashObject.reopen({ init() {} });
```

Comment:
`thought` `non-blocking`: I've noticed a few issues arise from calling `reopen` on an `EmberObject`. Thankfully, this is officially an adopted Ember add-on.

https://www.pzuraq.com/blog/emberjs-native-class-update-2019-edition#avoiding-reopen-and-reopenclass